utilities_common/bgp_util.py

```diff
@@ -256,7 +256,7 @@ def display_bgp_summary(bgp_summary, af):
         af: IPV4 or IPV6
 
     '''
-    headers = ["Neighbhor", "V", "AS", "MsgRcvd", "MsgSent", "TblVer",
```

Comment:
This is a legacy command, and many external monitoring tools would be parsing this field. I dont think this format can be changed now and suggest retaining it as it is.